OpenTelemetry.AutoInstrumentation.sln

```diff
@@ -151,7 +151,9 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "TestApplication.AspNet.NetF
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "TestApplication.MultipleAppDomains.NetFramework", "test\test-applications\integrations\TestApplication.MultipleAppDomains.NetFramework\TestApplication.MultipleAppDomains.NetFramework.csproj", "{A0338846-2AF8-4D96-ADA7-EBD4F010D08F}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "TestApplication.Quartz", "test\test-applications\integrations\TestApplication.Quartz\TestApplication.Quartz.csproj", "{B48780CE-8AAB-44D3-9CD1-8491584352B5}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "TestApplication.Quartz", "test\test-applications\integrations\TestApplication.Quartz\TestApplication.Quartz.csproj", "{B48780CE-8AAB-44D3-9CD1-8491584352B5}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "OpenTelemetry.AutoInstrumentation.Loader.Benchmark", "test\OpenTelemetry.AutoInstrumentation.Loader.Benchmark\OpenTelemetry.AutoInstrumentation.Loader.Benchmark.csproj", "{B1E71654-E784-4A39-BE45-8671E027B6D7}"
```

Comment:
I would move benchmarks to separate folder like in SDK repository https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/test/Benchmarks

All such tests should build in CI, but any of them should not be executed. Could you please confirm that it is the case?

Reply:
I've added it in the `_build` project. I'll confirm once CI run completes.